installation/touchegg.service.in

```diff
@@ -5,7 +5,7 @@ Documentation=https://github.com/JoseExposito/touchegg/tree/master/installation#
 [Service]
 Type=simple
 Group=input
-ExecStart=/usr/bin/touchegg --daemon
+ExecStart=${CMAKE_INSTALL_BINDIR}/touchegg --daemon
```

Comment:
By convention, at-sign delimited placeholders are used in `.in` files. Makes it less awkward if e.g. some shell is introduced.

```ini
ExecStart=@CMAKE_INSTALL_BINDIR@/touchegg --daemon
```

Reply:
Definitely! Fixed in f9a8849